src/others/xml/lv_xml.c

```diff
@@ -202,6 +211,35 @@ void * lv_xml_create_in_scope(lv_obj_t * parent, lv_xml_component_scope_t * pare
     }
 #endif
 
+    /*Create the timelines as well*/
+    if(!lv_ll_is_empty(&scope->timeline_ll)) {
+        lv_xml_timeline_t * at_xml;
+        lv_anim_timeline_t ** timeline_array;
+        timeline_array = lv_malloc((lv_ll_get_len(&scope->timeline_ll) + 1) * sizeof(lv_anim_timeline_t));
```

Comment:
```suggestion
        timeline_array = lv_malloc((lv_ll_get_len(&scope->timeline_ll) + 1) * sizeof(lv_anim_timeline_t *));
```

Allocation is much larger than it should be.